src/calendar/day/marking/index.tsx

```diff
@@ -41,6 +41,7 @@ export interface MarkingProps extends DotProps {
   marked?: boolean;
   today?: boolean;
   disabled?: boolean;
+  inactive?: boolean;
```

Comment:
Where are you passing 'inactive' to the Marking component? I don't see it in BasicDay...

Reply:
The type of BasicDay's `marking` is `MarkingProps` from the `Marking` component, so I was had to add it for using `this.marking.inactive` in BasicDay

Reply:
That's great, but you're not passing this prop in BasicDay component (here: https://github.com/wix/react-native-calendars/blob/master/src/calendar/day/basic/index.tsx#L170)

Reply:
You need to pass to the marking in order to style the marking according to the inactive state as you do in the Day component, no?

Reply:
I'm not sure, there's another inactive marking besides the inactive text color?

Reply:
There is no inactive marking, that's what I'm talking about. What will be the case of an inactive day with marking? How would it look?

Reply:
I see.. Added a support for that :)

Reply:
First, there are other marking beside Dot. Second, how will the inactive state get to the Marking and the Dot components if you're not passing them from BasicDay to the Marking?

Reply:
fixed